backend/cellguide/pipeline/gpt_descriptions/gpt_description_generator.py

```diff
@@ -0,0 +1,52 @@
+import logging
+import os
+
+from backend.layers.thirdparty.openai_provider import OpenAIProvider
+from backend.layers.thirdparty.s3_provider import S3Provider
+
+logger = logging.getLogger(__name__)
+
+
+def generate_all_gpt_descriptions(all_cell_type_ids_in_corpus: list[str]) -> None:
+    """
+    Generates GPT descriptions for all cell type ID's in the corpus if they haven't
+    already been generated. Cell type ids should be stored in the cellguide-data-public-<env>
+    bucket in the gpt_descriptions folder. For each cell type ID, we'll have a JSON file
+    that contains the GPT description
+
+    📁 gpt_descriptions
+    ﹂CL_0000000.json
+    ﹂CL_0000001.json
+    ﹂...
+
+    Returns a mapping of cell type id to GPT description
+    """
+
+    # TODO: figure out how to actually configure this environment variable
+    bucket_name = os.getenv("CELL_GUIDE_DATA_BUCKET_NAME")
```

Comment:
what's the recommended way to store bucket names? as environment variables or somewhere else?

Reply:
Check the PR I'm working on - there's a CellGuideConfig object I added which can retrieve the bucket stored in an AWS secret. in fact, I would recommend building your PR off my branch (`cellguide-fe`).